LICENSE

```diff
@@ -1,4 +1,5 @@
-   Copyright 2016 Maintainers of stig-part-minimal-linux
+   Copyright 2016,2020 Maintainers of stig-part-minimal-linux
+   Copyright 2020 Striveworks Inc.
```

Comment:
I would rather not modify the LICENSE file like this. If you would like, we'd certainly support a CONTRIBUTORS section in the readme (or in a separate file), instead.

Reply:
I added a CONTRIBUTORS.md file (https://github.com/plus3it/spel/pull/439/commits/f424d362781cb40d51fa9085fda654c0a8cfdf84).

I wasn't sure what to put for you folks, so I just copied the copyright owner.

Reply:
I got a bit confused ... you're correct and you did revert the change to the License file, but I liked that you updated the copyright.  Although I expect a dash should be used between the two years?  In any case, if Plus3IT wants/needs to update the copyright, they can do so.  Thank you for following up.